citrination_client/views/data_view_builder.py

```diff
@@ -58,6 +63,61 @@ def add_raw_descriptor(self, descriptor):
         """
         self.configuration['descriptors'].append(descriptor)
 
+    def add_relation(self, inputs, outputs, relation_type='lolo', options=None):
+        """
+        Add a manual relation.  If no relations are manually added, Citrination will automatically generate
+        relations when the view is created.
+
+        :param inputs: Array of strings or a single string of descriptor key(s) of the relation inputs
+        :param outputs: Array of strings or a single string of descriptor key(s) of the relation outputs
+        :param relation_type: Kind of relation, currently only 'lolo' is supported
+        :param options: A RelationOptions object with optional per relation settings
+        """
+        relation_obj = {}
+
+        if isinstance(inputs, str):
+            relation_obj['inputs'] = [inputs]
+        elif isinstance(inputs, list):
+            if len(inputs) == 0:
+                raise ValueError("Inputs list must not be empty")
+            relation_obj['inputs'] = inputs
+        else:
+            raise ValueError("Unexpected type for inputs, expecting either a string or list of strings")
+
+        if isinstance(outputs, str):
+            relation_obj['outputs'] = [outputs]
+        elif isinstance(outputs, list):
```

Comment:
Can a single relation support multiple outputs?

Reply:
Ha! it makes no sense for lolo relations, for material law stuff it does, but that does not apply here.  Ill update to single value.